Makefile

```diff
@@ -363,53 +363,33 @@ clang_format_fix:
 cbindgen: remove_cbindgen generate_cbindgen
 
 remove_cbindgen:
-	for h in components-rs/ddtrace.h components-rs/telemetry.h components-rs/sidecar.h components-rs/common.h; do if [ -f "$$h" ]; then rm "$$h"; fi; done
+	rm -f components-rs/ddtrace.h components-rs/telemetry.h components-rs/sidecar.h components-rs/common.h
 
-generate_cbindgen: components-rs/ddtrace.h components-rs/telemetry.h components-rs/sidecar.h components-rs/common.h
+generate_cbindgen: cbindgen_binary # Regenerate components-rs/ddtrace.h components-rs/telemetry.h components-rs/sidecar.h components-rs/common.h
```

Comment:
Some context:
Currently in our dev Docker container, running `make cbindgen` deletes  the `components-rs/*.h` files. 
Then, because the `cbindgen` command is not installed, the generation of the files is skipped silently.
Finally it fails in the deduplication step because the files are missing.

As the `components-rs/*.h` files cannot be generated individually because their declarations need to be deduplicated, I believe it's simpler to generate them in the same Makefile target.